test-configuration/base_config.inc

```diff
@@ -1,17 +1,16 @@
-driver_dir := ../../
 mux_dir := ../i2c-mux-testbed
-test_common_dir := ../embedded-common/test-config
+test_common_dir := ../embedded-common/test-configuration
```

Comment:
What do you think about this suggestion?

```suggestion
test_common_dir := ../embedded-common/test-config-common
```

Reply:
test-common-config? ... it's hard to name that stuff ... or tests-common-config? I surely like the common I only would put it in front of the config.

Reply:
works for me. My reasoning was that I just think it needs to start with test if its for testing and contain common if it's shared, config is equivalent to configuration so I'd go with the shorter.